tekton/cronjobs/bases/catalog/trigger-resource-cd.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: catalog-publish-trigger
+spec:
+  schedule: "12 * * * *"  # Houly at *:12
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+          - name: workspace
+            emptyDir: {}
+          containers:
+          - name: trigger
+            image: curlimages/curl
+            command:
+              - /bin/sh
+            args:
+              - -ce
+              - |
+                cat <<EOF > /workspace/post-body.json
+                {
+                  "trigger-template": "catalog",
+                  "params": {
+                    "git": {
+                      "repository": "$GIT_REPOSITORY",
+                      "revision": "$GIT_REVISION"
+                    },
+                    "catalog": {
+                        "registry": "$REGISTRY_URL",
+                        "registryPath": "$REGISTRY_PATH",
+                        "description": "$CATALOG_DESCRIPTION"
+                    }
+                  }
+                }
+                EOF
+                curl -d @/workspace/post-body.json $SINK_URL
```

Comment:
Good catch. The triggers sink does not seem to care about the content type, so this works fine, but I will set the extra header. This will have to be factored in all the existing cronjobs too 😅 

About the `curl -d @- <<EOF` I think that was my first approach, and then went back to the file to help with the initial troubleshooting. I guess I forgot to go back to `-d @-` and then it was copy pasted away :)

Reply:
All fixed